google/ggrc-core
# GGRC-5199  Apply same styles for all unmap buttons

# Issue description

Reference URLs and Files have different opacity of Unmap button.

# Steps to test the changes

1. Add Reference Url to an object
2. Add file to the same object
3. Check 'unmap' buttons color.

**Expected result:** Styles are the same.
**Actual result:** Styles are different.

# Solution description

Use the same styles for all unmap buttons.

# Sanity checklist




# PR Review checklist
